Products/LoginLockout/plugin.py

```diff
@@ -303,9 +303,19 @@ def isLockedout(self, login):
 
     security.declarePrivate('isIPLocked')
     def isIPLocked(self, login, IP):
+        try:
+            whitelist_ips = self._whitelist_ips
+        except AttributeError:
+            # The attribute is not there
+            return False
+
+        if not whitelist_ips:

```

Comment:
Generally you should do len(whitelist_ips) == 0 since "not" can have unintended consequences.